packages/snaps-jest/src/global.ts

```diff
@@ -42,16 +43,21 @@ interface SnapsMatchers {
    * `expect(response.notifications).toContainEqual({ message, type })`.
    *
    * @param message - The expected notification message.
-   * @param type - The expected notification type, i.e., 'inApp' or 'native'. If
-   * not provided, the type will be ignored.
+   * @param type - The expected notification type, i.e., 'inApp' or 'native'.
+   * @param title - The title of an expanded notification.
+   * @param content - The content of an expanded notification.
+   * @param footerLink - The footer link of an expanded notification (if it exists).
    * @throws If the snap did not send a notification with the expected message.
    * @example
    * const response = await request({ method: 'foo' });
    * expect(response).toSendNotification('bar', NotificationType.InApp);
    */
   toSendNotification(
     message: string,
-    type?: EnumToUnion<NotificationType>,
+    type: EnumToUnion<NotificationType>,
```

Comment:
Was this intentional? If so, it's a breaking change.
```suggestion
    type?: EnumToUnion<NotificationType>,
```

Reply:
Yes, intentional.

Reply:
It feels weird to allow something like `.toSendNotification('hello', undefined, 'title', <Component>)`

Reply:
Not a hill I'd die on, but if we're talking about the devx being great, this makes it weirder. Also not sure how much this API is in current use?

Reply:
I'd prefer making it not breaking for now. We can revisit this API in general later.